src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Legacy/CSharpCodeParser.cs

```diff
@@ -2008,14 +2011,13 @@ private bool TryParseKeyword(in SyntaxListBuilder<RazorSyntaxNode> builder)
 
     private bool AtBooleanLiteral()
     {
-        var result = CSharpTokenizer.GetTokenKeyword(CurrentToken);
-        return result.HasValue && (result.Value == CSharpKeyword.True || result.Value == CSharpKeyword.False);
+        return LegacyCSharpTokenizer.GetTokenKeyword(CurrentToken) is CSharpSyntaxKind.TrueKeyword or CSharpSyntaxKind.FalseKeyword;
```

Comment:
What is LegacyCSharpTokenizer?

Reply:
It's sign I missed one thing when fixing up the cherry-pick